src/Courses/TopicSettings/ProblemSettings.tsx

```diff
@@ -225,111 +238,125 @@ export const ProblemSettings: React.FC<ProblemSettingsProps> = ({selected, setSe
         }
     };
 
+    const disabled = saving || topic.retroStartedTime !== null;
     return (
         <FormProvider {...topicForm}>
             <HasEverBeenActiveWarning topic={topic} />
             <PromptUnsaved message='You have unsaved changes. Are you sure you want to leave the page?' when={formState.isDirty} />
             <form onChange={() => {if (updateAlertMsg !== '') setUpdateAlert({message: '', severity: 'warning'});}} onSubmit={handleSubmit(onSubmit)}>
-                <DevTool control={control} />
-                <Grid container item md={12} spacing={3}>
-                    <Snackbar
-                        anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
-                        open={updateAlertMsg !== ''}
-                        autoHideDuration={updateAlertType === 'success' ? 6000 : undefined}
-                        onClose={() => setUpdateAlert(alertState => ({...alertState, message: ''}))}
-                        style={{maxWidth: '50vw'}}
-                    >
-                        <MUIAlert
+                <fieldset disabled={disabled}>
```

Comment:
You could pass `disabled` down to the individual MUI components to get the styling right.